### DISCUSSION TITLE
2022-06: Iteration 6 汇报

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#333] [Announcements]
# 2022-06: Iteration 6 汇报

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-06/

**Comment by STRRL** [comment]
饮水机出水速度怎么样呀？比如说 350ml 大约要多久？

**Reply by Xuanwo** [reply]
专门去掐了个秒表测了一下：

- 1 秒开始出热水
- 从按键开始到 350ml 出水完毕花了 49 秒

**Reply by pymongo** [reply]
既热饮水机不存储热水看起来不错，我之前的解决方案是矿泉水倒入小米 199 那个烧水壶，300ml 比你快点设置加热成 40 度大约 20 秒
水凉了之后再加水加热到 40 度就行

**Reply by pymongo** [reply]
sync/async 之间交互方便的办法我感觉是在 Context 结构体上放一个单线程 tokio runtime 的字段
这样就能在 Drop 函数里面异步发 tonic rpc 请求给 server 回收当前 session 之类的资源

**Comment by STRRL** [comment]
俺现在喝水的方案是这样： ![image](https://user-images.githubusercontent.com/20221408/153715575-6979465b-9f25-4858-81d4-38e84d18e5d7.png)

（350ml 大约需要 6 秒， 问题就是 buffer 太小， 只有 1.5L
